hooks/queries/useFetchConcerts.ts

```diff
@@ -0,0 +1,10 @@
+import { useQuery } from '@tanstack/react-query';
+
+import { getContertDetail } from '~/apis/performance';
+
+export const useFetchConcerts = (id: string) => {
+  return useQuery({
+    queryKey: ['concert', id],
+    queryFn: () => getContertDetail(id),
```

Comment:
```suggestion
    queryFn: () => getContertDetails(id),
```
이렇게 해주시면, 위에 getConcertReviews 와 통일될것 같아요!🙂